tests/custom_procedure_playground.html

```diff
@@ -193,8 +204,14 @@
           addBoolean();
           break;
       }
+    }
 
-      applyMutation();
+    function cancel() {
+      callback = null;
+      mutationRoot = null;
+      declarationWorkspace.clear();
+      definitionWorkspace.refreshToolboxSelection_()
```

Comment:
No need to refresh the toolbox selection here.

Reply:
That is done because the procedures category needs to be updated because of the change in blocks, and mutateCallers doesn't reach into the toolbox workspace to change those. Should it?

Reply:
Good point.  This is fine.